qcelemental/covalent_radii.py

```diff
@@ -21,20 +21,24 @@ class CovalentRadii:
 
     Attributes
     ----------
-    name : str
-        The name of the context ('ALVAREZ2008')
     cr : dict of Datum
         Each covalent radius is an entry in `cr`, where key is the
         "Fe"-cased element symbol if generic or symbol-prefixed label
         if specialized within element. The value is a Datum object with
         `lbl` the same as key, `units`, `data` value as Decimal object,
         and any uncertainty in the `comment` field.
+    doi : str
+        The DOI of the current context.
+    name : str
+        The name of the context ('ALVAREZ2008')
+    native_units : str
+        The units the original data was provided in.
     year : int
         The year the context was created.
 
     """
 
-    def __init__(self, context="ALVAREZ2008"):
+    def __init__(self, context: str="ALVAREZ2008"):
```

Comment:
This syntax need to be `context: str = "ALVAREZ2008"` to follow YAPF.

Reply:
Hmm, I thought this was a trick question.